src/monthly-giving-circle.ts

```diff
@@ -1,22 +1,104 @@
 /* eslint-disable no-debugger */
 
-import { LitElement, html } from 'lit';
+import { LitElement, html, TemplateResult, nothing } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
 import './welcome-message';
 import './presentational/mgc-title';
+import './receipts';
+import type { IauxMgcReceipts } from './receipts';
+import './presentational/iaux-button';
+
+export type anUpdate = {
+  message: string;
+  status: 'success' | 'fail';
+  donationId: string;
+};
 
 @customElement('iaux-monthly-giving-circle')
 export class MonthlyGivingCircle extends LitElement {
   @property({ type: String }) patronName: string = '';
 
+  @property({ type: Array }) receipts = [];
```

Comment:
This can become
```
@property({ type: Array }) receipts: aReceipt[] = [];
```
so TS knows the type of data its working with and help the developer out during development.